open-media-match/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -106,12 +107,7 @@ def lookup():
     return enabled_banks
 
 
-@bp.route("/index/status")
-def index_status():
-    """
-    Input:
-     * Signal type (hash type)
-    Output:
-     * Time of last index build
-    """
```

Comment:
blocking: Bring back this docstring or something like it

Reply:
Restored